services/app/src/components/avatarPicture/DefaultAvatar.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Avatar } from 'antd';
+import { getAvatarPxSize, getAvatarFontSize } from '../../helpers/getAvatarSizes';
+import { useChessBoardContext } from '../../contexts/ChessBoardContext';
+import { rgbToColor } from '../../helpers/rgbToColor';
+
+const DefaultAvatar = ({ size, id, isBorderNeeded }) => {
+  const { findUserById } = useChessBoardContext();
```

Comment:
Check if you could use currentplayer instead of passing id

Reply:
No, we can not, because this component also renders team's avatars, so currentPlayerId would not work in this case